main.go

```diff
@@ -11,6 +11,6 @@ var log = logging.MustGetLogger("docker-on-top")
 func main() {
 	log.Info("Hello there!")
 
-	handler := volume.NewHandler(&Driver{volumesCreated: map[string]bool{}})
+	handler := volume.NewHandler(&Driver{volumesCreated: map[string]volume.Volume{}, overlaysCreated: map[string]string{}})
```

Comment:
It would be better if the plugin was capable of restoring the volumes list on restart (by examining its internal directory). But I think this could be implemented outside of this PR